### DISCUSSION TITLE
Submitting .pdf report instead of .md

### DISCUSSION THREAD

**Discussion by GitianOberhuber** [sebastian-hofstaetter/teaching#108] [Advanced IR: Exercise Q&A] [answered]
Hi,
We finished our report (in a Google Docs document, which we planned to export as .pdf) and only now realized that the report-file "Report.md" in the repository is markdown file. Is it ok if we submit a pdf report instead of a markdown report?

**Comment by sophiaalthammer** [comment, marked as answer]
Hi,
yes a pdf file is also ok!

Best,
Sophia
